src/Sylius/Component/Core/Model/AdminUserInterface.php

```diff
@@ -14,8 +14,9 @@
 namespace Sylius\Component\Core\Model;
 
 use Sylius\Component\User\Model\UserInterface as BaseUserInterface;
+use Symfony\Component\Security\Core\User\EquatableInterface;
 
-interface AdminUserInterface extends BaseUserInterface, ImageAwareInterface
+interface AdminUserInterface extends BaseUserInterface, ImageAwareInterface, EquatableInterface
```

Comment:
It seems to be a bc-break, no? Could we just implement the Interface on AdminUser

Reply:
Model interfaces are excluded 💃🏻! 
.
.
.
Being serious, you're right. I'll move it :D.

Reply:
Fixed